pull_request_template.md

```diff
@@ -1,8 +1,16 @@
+## Ticket
+Please include a link to the ticket
+
 ## Description
 Please include a summary of the changes
 
-## Ticket
-Please include a link to the ticket
+## Type of change
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Non-prod (documentation update, etc.)
+
+## How to Test (steps to reproduce changes if needed)
```

Comment:
Would it be useful to include non-header text to get the developer started on writing this section? If nothing else, it might subtly push for consistent formatting.

Something like:

```
## How to Test
1. Include a list of steps to verify the changes
```